aws-logs-metricfilter/src/test/java/software/amazon/logs/metricfilter/AbstractTestBase.java

```diff
@@ -1,8 +1,11 @@
 package software.amazon.logs.metricfilter;
 
-import java.util.Arrays;
+import java.util.*;
```

Comment:
np: we import `java.util.*` here then import `java.util.stream.Collectors`, may be better to just avoid using import *

Reply:
agreed, updating imports